sinon/lib/util/Wrapper.py

```diff
@@ -77,7 +77,7 @@ def wrapped(*args, **kwargs):
             # Todo: make sure e.__class__ is enough for all purpose or not
             wrapped.error_list.append(excpt.__class__)
             call.exception = excpt
-            return func(*args, **kwargs)
+            raise excpt
```

Comment:
Thanks for pointing out this, this make the action as same as SinonJS.

(Just note: Since I forget the reason why I wrote the code in this way to bypass exception, I am not sure there is any issue after this change.)

Reply:
You're welcome. I think it's important to change the code to raise the original exception. Imagine if the stubbed function were like this:

```python
def my_function(self):
    self.num_times_run += 1
    if self.num_times_run == 1:
        raise BaseException("Failed on first run!")
    if self.num_times_run > 1:
        raise Exception("Failed during later run")
```
With the above function, it raises a different exception each time it is run. 

I'm not sure, but there could also be an issue with the call count if you re-call the function a second time.

Reply:
You are right, this point was not considered well in previous opinion.